### ISSUE TITLE
Node.js Foundation Technical Steering Committee (TSC) Meeting 2019-05-29

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#713]
## Time

**UTC Wed 29-May-2019 16:00 (04:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 29-May-2019 09:00 (09:00 AM) |
| US / Mountain | Wed 29-May-2019 10:00 (10:00 AM) |
| US / Central  | Wed 29-May-2019 11:00 (11:00 AM) |
| US / Eastern  | Wed 29-May-2019 12:00 (12:00 PM) |
| London        | Wed 29-May-2019 17:00 (05:00 PM) |
| Amsterdam     | Wed 29-May-2019 18:00 (06:00 PM) |
| Moscow        | Wed 29-May-2019 19:00 (07:00 PM) |
| Chennai       | Wed 29-May-2019 21:30 (09:30 PM) |
| Hangzhou      | Thu 30-May-2019 00:00 (12:00 AM) |
| Tokyo         | Thu 30-May-2019 01:00 (01:00 AM) |
| Sydney        | Thu 30-May-2019 02:00 (02:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2019-05-29&iso=20190529T16
* or http://www.wolframalpha.com/input/?i=04PM+UTC%2C+May+29%2C+2019+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1hMPPigBlhTNR_GQULz9A1RB05NAz7c8mMKMrnK1J7q4/edit>

## Agenda

Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* benchmark: refactor buffer benchmarks [#26418](https://github.com/nodejs/node/pull/26418)

### nodejs/TSC

* Reevaluate meeting times [#707](https://github.com/nodejs/TSC/issues/707)
* doc: update TSC charter [#698](https://github.com/nodejs/TSC/pull/698)
* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)

## Invited

* Anna Henningsen @addaleax (TSC)
* Anatoli Papirovski @apapirovski (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Daniel Bevenius @danbev (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Jeremiah Senkpiel @Fishrock123 (TSC)
* Gabriel Schulhof @gabrielschulhof (TSC)
* Gireesh Punathil @gireeshpunathil (TSC)
* James Snell @jasnell (TSC)
* Joyee Cheung @joyeecheung (TSC)
* Matteo Collina @mcollina (TSC)
* Michael Dawson @mhdawson (TSC)
* Myles Borins @MylesBorins (TSC)
* Sam Roberts @sam-github (TSC)
* Michaël Zasso @targos (TSC)
* Sakthipriyan Vairamani @thefourtheye (TSC)
* Rich Trott @Trott (TSC)

## Notes

The agenda comes from issues labelled with `tsc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Zoom link: https://zoom.us/j/611357642
Regular password

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the TSC that's not worth putting on as a separate agenda item, this is a good place for that.

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

**Comment by mcollina** [comment]
Some of us will be traveling to Berlin. I propose to cancel.

**Comment by mhdawson** [comment]
+1 to cancelling, I don't think there is anything new/urgent this week and as @mcollina mentioned many people will be travelling at this time (myself included).

**Comment by Trott** [comment]
I'm good with canceling but I'd also like us to take this opportunity to push forward the two items on the agenda (not counting the standing items). There's no need for a meeting to do this stuff, but there is a need for TSC involvement. I'm going to put what I *think* is a decent way forward for each of those items in subsequent comments.

**Comment by Trott** [comment]
For https://github.com/nodejs/node/pull/26418, @sam-github seems to have clear ideas and definite opinions about the *way* that disagreement should be resolved, if not about what that resolution will ultimately look like. Sam, would you be willing to take an action item to review the most recent few comments, and formulate a way forward? That could be proposing "let's do X" and then seeing if you can get buy-in on X from @BridgeAR and @mscdex, modifying X as necessary. Or it could be *not* having a preconceived solution X and reaching out directly and privately (email?) to @mscdex and @BridgeAR to try to work something out that is agreeable to both parties. Or whatever you like. Just as long as something is being done to inch it towards a resolution one way or the other. The alternatives, as I see it, are to hold a TSC vote, or simply close the issue. I'm OK with a vote if we think we're at an intractable impasse. In that case, I guess the action item is to cajole TSC members who haven't participated in the vote yet to do so (even if it's just to explicitly abstain). In any event, can we assign that to you?

**Comment by Trott** [comment]
For https://github.com/nodejs/TSC/issues/707, I think we should approve the times @ChALkeR has calculated as best/most-fair/etc.

TL;DR: If you're a TSC member, 👍 this comment to approve UTC 15 16 19. Use 👎 if you think those times are bad/unfair for the TSC as a whole, but please leave a comment/show your work. Use 😕if you don't have an opinion but trust everyone else to figure it out.

My take: @ChALkeR has done great work extending the workable-but-simplistic algorithm that we used to use into something a fair bit more sophisticated. This is a hard problem and whatever time slots we choose, there will be issues. I'm confident that the ones he's suggesting are among the least problematic. I would encourage rubber-stamping this and dissuade people from proposing alternatives without doing a *lot* of looking into the factors that go into this. This is deceptively tricky stuff and I know I used to get frustrated after spending a *lot* of time looking at things, only to have people drive-by with suggestions that weren't thought through.

**Comment by mhdawson** [comment]
@Trott thanks for pushing on #707.  Lets try to get enough people to comment so that we can adjust for the week of the 3rd as it seems that we would be moving away from the 12 UTC timeslot which is currently what the 3rd is set for.

**Comment by mhdawson** [comment]
No objections so cancelling the meeting for tomorrow.
